### PR TITLE
remuv mirror appearence change

### PR DESCRIPTION
**Pull request by jughu**
WE HAVE A PERFECT BARBER NOW TO DO THIS kind of STUFF

### CODE REVIEW

**Comment by Necaladun** [comment]
The barber is not available...at all, really, and the mirror change is useful for a lot of other things. It's also used in many places like the wizards den, centcomm, etc.

**Comment by jughu** [comment]
dont wizards get the magic variant

**Comment by tigercat2000** [comment]
No. The barber is almost never around, this just fucks people over.

**Comment by Aurorablade** [comment]
You know if people would ask for the barber when they see them in the manifest an if we limited the mirror it might encourage people to seek them out but eh

**Comment by SamHPurp** [comment]
Limit the mirror, swap the Janitors closet with the Barber shop. More RP with no real negatives.

**Comment by Fox-McCloud** [comment]
Locking a mechanic behind a job that is karma only is generally bad form (and before someone says "what about Mechanic", yes, Mechanic is an example of "bad form" in a karma job).

It's be one thing if this job didn't require karma, but it does.

**Comment by jughu** [comment]
barber is more a bait trap to waste karma

**Comment by Fox-McCloud** [comment]
>barber is more a bait trap to waste karma

Everyone knows it's greytide+ 😏

**Comment by jughu** [comment]
scissors are fucking robust im tellin ya

**Comment by taukausanake** [comment]
I'm in favor of this but not by itself. It's been mentioned before to add a feature to barbering and I've had it in my mind too for a while. Set hair styles with a hair_length var and mirrors can change hairstyles into those with the same length. Only the barber can change that length though the use of scissors or hair growth tonic stuff along with styling it too

**Comment by Dumbdumn5** [comment]
But why not both?

File: code/game/objects/structures/mirror.dm [hunk 1]
```diff
@@ -12,14 +12,6 @@
 /obj/structure/mirror/attack_hand(mob/user as mob)
```
Comment (KasparoVy) [inline code/game/objects/structures/mirror.dm]: While I don't support this, if you're going to axe it you might as well remove the entire attack_hand section as it exists solely for the appearance changing.

**Comment by Fox-McCloud** [comment]
Locking something like this behind a karma job is a no-no...especially something that was a mechanic that was previously available to everyone.

It would be like turning a non-karma race into a karma race.

**Pull request closed without merging**